subsys/bootloader/Kconfig

```diff
@@ -26,7 +26,7 @@ config SECURE_BOOT_CRYPTO_CLIENT
 config SECURE_BOOT
 	bool "Use Secure Bootloader"
 	depends on !IS_SECURE_BOOTLOADER
-	select SECURE_BOOT_VALIDATION
+	select FW_INFO
```

Comment:
depends on?

Reply:
Why do we need this select now, how was this solved before? Did the bootloader cmake code inject the kconfig to the application?

Reply:
Changed to `depends on`